acrobat/blocks/app-banner/app-banner.js

```diff
@@ -0,0 +1,128 @@
+import { createTag } from "../../scripts/miloUtils.js";
+
+var link = '';
+var mobileOS = getMobileOperatingSystem();
+
+
+export default async function init(el) {
+  //Exit if not mobile/tablet 
```

Comment:
this is apparent we don't need it called out

Reply:
Removed.